package.json

```diff
@@ -2,17 +2,16 @@
   "name": "ifrau",
   "version": "0.37.0",
   "description": "Free-range app utility for IFRAME-based FRAs",
-  "main": "src/index.js",
+  "type": "module",
```

Comment:
I've also seen packages use `"module": "index.js"` where `index.js` then exports all the things that you can import using `import { foo, bar } from 'ifrau';`.

Reply:
Do we have any particular preference here? I can easily do that as well. I'm not super sure what's more common/useful/preferred in the long run. Perhaps your suggestion because the package can be referenced directly without needing to wander into subdirectories to pull out the right exports.

Reply:
Yeah I'd prefer it only export the things that it wants you to be able to import, which is what that `module` entry is for.